File: 1pac/1.4-Algoritmos-File y Random/RandomCSV.py
import random
import math

def random2Int(max = 100, min = 0):
    myInt = int(random.random()*(max-min)+min)
    myInt= math.floor(myInt)

    return myInt
 
class RandomCSV:
    
    def generateCSV(self):
        c = random2Int(15,10)
        r = random2Int(15,10)
        rows = []
        for i in range(0,c,1):
            columns = []
            for j in range(0,r,1):
                columns.append(str(random2Int(10,100)))
            
            rows.append(",".join(columns))

        return "\n".join(rows)




    

File: 1pac/1.04-Algoritmos-File y Random/RandomCSV.js
function RandomCSV(){

    this.generate = RandomCSVGenerate;
}

    function RandomCSVGenerate(){
        r=Random2Int(10,20);
        c=Random2Int(10,20);
        c=10;
        row = [];
        
        for(j=0;j<r;j++){
            columns = [];

            for(i=0;i<c;i++){
                columns.push(Random2Int());    
            }
            row.push(columns.join(","));              
        }

        return row.join("\n");
    }